_data/projects/CommonIntents.yml

```diff
@@ -0,0 +1,10 @@
+name: CommonIntents
+desc: Android library that provides most common android intents in a convenient manner
+site: https://github.com/JanStoltman/CommonIntents
+tags:
+- android
+- kotlin
+- mobile
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/JanStoltman/CommonIntents/issues
```

Comment:
@JanStoltman Please mention the link to the issues related to your label here, Also do add some issues so that others could contribute.